newdle/client/src/components/MyNewdles.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {Container, Icon, Label} from 'semantic-ui-react';
+import {serializeDate, toMoment} from '../util/date';
+import {useRouter} from '../util/router';
+import styles from './MyNewdles.module.scss';
+
+function MyNewdle({newdle}) {
+  const router = useRouter();
+  return (
+    <div
+      className={styles.newdle}
+      key={newdle.id}
+      onClick={() => router.history.push(`/newdle/${newdle.code}/summary`)}
+    >
+      <h3 className={styles.title}>
+        {newdle.title}
+        <Label color={newdle.finalDt ? 'blue' : 'green'} size="tiny" className={styles.state}>
+          {newdle.finalDt ? 'finished' : 'ongoing'}
+        </Label>
+      </h3>
+      {newdle.finalDt && (
+        <div className={styles.info}>
+          <span>
+            <Icon name="calendar alternate outline" />
+            <label>{serializeDate(newdle.finalDt, 'Do MMMM')}</label>
+          </span>
+          <span>
+            <Icon name="clock outline" />
+            <label>
+              {`${serializeDate(newdle.finalDt, 'HH:mm')} - ${serializeDate(
```

Comment:
Now that it's a separate content you could do the formatting before getting into the return jsx block, and thus avoid the awful wrapping here:

```
const startDate = ...;
const endDate = ...;
```